gcn/notices/km3net/test/medal_ranking_alert.example.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/km3net/test/medal_ranking_alert.schema.json",
+  "mission": "KM3NeT",
+  "instrument": "ARCA028",
+  "messenger": "Neutrino",
+  "packet_type": 999,
+  "alert_tense": "test",
+  "alert_type": "initial",
+  "alert_datetime": "2024-09-01T12:01:00.00Z",
+  "analysis_pipeline": "exceptional_evt_arca",
+  "description": "KM3NeT online analysis, bronze candidate neutrino observation.",
```

Comment:
Make it specific field, like medal_rank, instead of description. It is useful property, should be machine-readable.